### ISSUE TITLE
reuseBucketQuads causes strange glitch with buckets and AE Additions patterns

### ISSUE THREAD

**Issue by embeddedt** [LoliKingdom/LoliASM#55]
When [AE Additions](https://www.curseforge.com/minecraft/mc-mods/ae-additions-extra-cells-2-fork) and LoliASM are installed, Forge universal buckets seem to have their overlay texture overwritten by the one from AE Additions' patterns, if `reuseBucketQuads` is enabled.

This is a minor issue. It seems to have a bit of a race condition going on, as it does not always occur depending on what items you render in game first. For that reason, I also had difficulty reproducing it in a clean instance with just the essential mods installed. I can reliably reproduce it in a specially crafted world in the full modpack, however.

With `reuseBucketQuads` disabled:

![image](https://user-images.githubusercontent.com/42941056/144161529-cd7496bb-2998-4e9a-8ee1-f3785f0b88b9.png)

With `reuseBucketQuads` enabled:

![image](https://user-images.githubusercontent.com/42941056/144162309-9cf7b5fc-ae88-44bb-924c-a6c87b845a08.png)

AE Additions source code: https://github.com/the9grounds/AE-Additions/tree/master

**Comment by Rongmario** [comment]
Yeah, I wanted to debug this a while back actually. Thanks for making an issue here, unsure when I'll get around to fixing it.

**Comment by Krutoy242** [comment]
Experienced same

![](https://i.imgur.com/iv4zYVf.png)

**Comment by Krutoy242** [comment]
Issue was fixed for buckets, but still persist for other liquid containers:
![image](https://user-images.githubusercontent.com/936618/165338715-805956b5-a7b8-469b-9fa4-3732983c25bc.png)
[debug.log](https://github.com/LoliKingdom/LoliASM/files/8564694/debug.log)

**Comment by Rongmario** [comment]
I don't have those issues on 4.11+?
![image](https://user-images.githubusercontent.com/3684700/165341511-7fd467fc-4443-44d7-8a6a-09bf32c58fdc.png)

**Comment by Rongmario** [comment]
Ah you're using 4.10.5, it was fixed in 4.11.

**Comment by Krutoy242** [comment]
Damn, sorry for bothering you with invalid issues again...
